addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2398,7 +2399,7 @@ export class Wysiwyg extends Component {
                 callback: async () => {
                     const [user] = await this.orm.read(
```

Comment:
@shsa-odoo As @aab-odoo mentioned, it would be best to rename this so as not to shadow the `user` object.

Reply:
@shsa-odoo Friendly reminder.

cc @dmo-odoo

Reply:
Hello @Zynton i was away for few weeks. I will make the changes asap.

Reply:
@Zynton made changes, we are good to go.